### PR TITLE
Create ExpandingWrapper

### PR DESCRIPTION
**Pull request by sophie-macmillan**
This commit adds a new component called ExpandingWrapper. 

## What is the purpose of this change?
Adds a new component called ExpandingWrapper.

## What does this change do?
The component wraps any children, and collapses them by default, they can be fully shown by clicking the 'Show More' button, and then the 'Show Less' button once expanded.

This component will be used in the Callout component in both Apps Rendering, and Dotcom Rendering, and a similar expander is used for pinned posts, so this could potentially be updated to use the shared ExpandingWrapper in the future.

## Screenshots

**After**

https://user-images.githubusercontent.com/26366706/201893468-2d00ce73-c8dd-4ca8-9f0c-1c1218829d87.mov

### Accessibility

- [x] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
- [x] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
    - Button is aria-hidden, as the content is only visually collapsed - it is all accessible to a screenreader
- [x] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
- [x] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### Cross browser and device testing

- [x] Tested with touch screen device
    - Tested with voiceover on IOS, and for touch on an iphone

### Responsiveness

- [x] Tested at all breakpoints
- [x] Tested with with long text
- [x] Stretched to fill a wide container

### Documentation

- [x] Full API surface area is documented in the README
- [x] Examples in Storybook

### CODE REVIEW

**Review by ob6160 — DISMISSED** [review]
looks great, an exciting addition to the kitchen! 🙂

Github added this error to the codeowners file, not sure if it's accurate? Not a blocker, just thought I'd mention 
Screenshot 2022-11-10 at 10 07 09

**Comment by ob6160** [comment]
> > Github added this error to the codeowners file, not sure if it's accurate?
> 
> teams that are mentioned in the codeowners file need to be explicitly added to the repo, otherwise they will never be alerted to changes

ah thanks, I've granted @guardian/editorial-experience write access

**Comment by sophie-macmillan** [comment]
Thanks everyone! Sorry for the silence on this, I moved it back into draft while getting UX feedback and making those changes.

It's ready to go again now! Have resolved the semver issue, and looks like the `codeowners` file is valid after you granted us access thanks @ob6160!

**Review by ob6160 — APPROVED** [review]
looks great, thanks so much for the contribution! ⭐

File: packages/@guardian/source-react-components-development-kitchen/src/expanding-wrapper/ExpandingWrapper.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+ * [Storybook](https://guardian.github.io/source/?path=/docs/packages-source-react-components-label--playground) •
+ * [Design System](https://theguardian.design/2a1e5182b/p/40151e-label/b/86af7d) •
+ * [GitHub](https://github.com/guardian/source/tree/main/packages/@guardian/source-react-components/src/label/Label.tsx) •
+ * [NPM](https://www.npmjs.com/package/@guardian/source-react-components)
+ *
+ * An Expanding Wrapper surrounds a bit of content that can be collapsed and expanded
+ * Note: When collapsed, there should be no focusable elements within the wrapper.
+ *
+ * The following themes are supported: `light`.
+ * */
```
Comment (sndrs) [inline packages/@guardian/source-react-components-development-kitchen/src/expanding-wrapper/ExpandingWrapper.stories.tsx:17]: is this comment correct for this story? the links refer to `source-react-components/src/label`?

Comment (sophie-macmillan) [reply packages/@guardian/source-react-components-development-kitchen/src/expanding-wrapper/ExpandingWrapper.stories.tsx]: Right you are - have updated

**Review by ob6160 — APPROVED** [review]

**Pull request merged**